ini/adillor.ini

```diff
@@ -1,41 +1,41 @@
-; please byteswap your sample ROMs (am1wavea.2l and am1waveb.1l) if you have the MAME set
-; i will submit a pull some time later unless someone beats me to it...
-
-[data]
-name     = Armadillo Racing
-driver   = Quattro
-type     = SS22
-filename = am1data.8k
-gain     = 0.70
-muterear = 1
-
-[wave.0]
-filename = am1wavea.2l
-length   = 0x400000
-position = 0x0
-
-[wave.1]
-filename = am1waveb.1l
-length   = 0x400000
-position = 0x800000
-
-[playlist]
-; http://vgmdb.net/album/3065
-8d = Armadillos and Good Friends [Attract 1]
-8e = Begin - Armadillos and Good Friends [Attract 2]
-81 = The Way of the Armadillo [Select]
-80 = Swirl [Sprint Course]
-88 = ITAZURA [Obstacle Course]
-83 = Solar Beat [Race Finish]
-82 = Giddiness [Bonus Game]
-85 = Bonus Game Results
-8a = Red Blossom on a Cactus [Time-out]
-8b = The Skippy Way Back [Game Finish]
-89 = Colors of the Rainbow [1st Place Ending]
-8c = 2nd Place Ending
-c0 = 3rd~4th Place Ending
-8f = Shortcut Through the Woods [Name Entry]
-86 = unknown 1
-87 = unknown 2
-c3 = unknown 3
-84 = unknown 4
+; please byteswap your sample ROMs (am1wavea.2l and am1waveb.1l) if you have the MAME set
+; i will submit a pull some time later unless someone beats me to it...
+
+[data]
+name     = Armadillo Racing
+driver   = Quattro
+type     = SS22
+filename = am1data.8k
+gain     = 0.70
+muterear = 1
+
+[wave.0]
+filename = am1wavea.2l
+length   = 0x400000
+position = 0x0
+
+[wave.1]
+filename = am1waveb.1l
+length   = 0x400000
+position = 0x800000
+
+[playlist]
+; http://vgmdb.net/album/3065
+8d = Armadillo And His Friends [Attract 1]

```

Comment:
source on the translated track titles? The original ones were taken from vgmdb as mentioned in the file.

Reply:
The previous translations were a bit...off. So I re-translated them myself using DeepL and other sources.